src/components/CartButton/CartButton.css

```diff
@@ -0,0 +1,28 @@
+.red-badge {
+	height: 15px;
+	width: auto;
+	min-width: 11px;
+	padding-left: 2px;
+	padding-right: 2px;
+	white-space: nowrap;
+	border-radius: 8px;
+	background: #ff0000;
+	position: absolute;
+	top: 0px;
+	right: 0px;
+	margin-right: -3px;
+	margin-top: -3px;
```

Comment:
You can shorten this code by changing the top and right values to -3px and removing the margin-right and margin-top lines